### PR TITLE
install doc: more discussion on admin stuffs

### PR DESCRIPTION
**Pull request by matthewrmshin**
Move sections around so:
* user stuffs are nearer the top.
* easier stuffs are nearer the top.
* Rosie client configuration and Rosie server configuration are in separate sections.

Discuss whether it is possible/necessary to set up Rose Bush and/or
Rosie web service.

Updated `rose.conf.example` file so that `[section]` names appear at the top (as well as at the bottom). It looks a bit strange, but is syntax-OK, and users are less likely to get it wrong.

@kaday please review.

### CODE REVIEW

File: doc/rose-install.html [hunk 1]
```diff
@@ -346,14 +350,156 @@ <h2 id="rose">Configuring Rose</h2>
...
 
     <p>Rosie stores suites using Subversion repositories, with databases behind
-    a web interface for suite discovery and lookup. You will need to select a
-    machine to host the Subversion repositories. This machine will also host
-    the web server and databases.</p>
+    a web interface for suite discovery and lookup.</p>
+
+    <p>If users at your site are able to access Rosie services on the Internet
+    or if someone else has already configured Rosie services at your site, all
+    you need to do is configure the client to talk to them. Refer to the
```
Comment (kaday) [inline doc/rose-install.html]: Explain what is meant by "them." maybe something like "the Rosie services."

File: doc/rose-install.html [hunk 2]
```diff
@@ -346,14 +350,156 @@ <h2 id="rose">Configuring Rose</h2>
...
+    <a href="#rosie-server">Configuring a Rosie Server</a> section if you need
+    to configure a Rosie server for your site.</p>
+    
+    <p>To set up the Rosie client, add/modify the <code>[rosie-id]</code>
+    section of <samp>etc/rose.conf</samp> E.g.:</p>
```
Comment (kaday) [inline doc/rose-install.html]: Make clear if this is at the site or user level.

File: doc/rose-install.html [hunk 3]
```diff
@@ -346,14 +350,156 @@ <h2 id="rose">Configuring Rose</h2>
...
+prefix-web.myorg=http://myhost.myorg/trac/roses-myorg/intertrac/source:
+prefix-ws.myorg=http://myhost.myorg/rosie/myorg
+</pre>
+
+    <p>Check the following:</p>
+
+    <ol>
+      <li>You can access the Rosie Subversion repository without being prompt
+      for a username and a password. This may require configuring Subversion to
+      cache your authentication information with a keyring.</li>
```
Comment (kaday) [inline doc/rose-install.html]: Add a link of how to do this or where the information is on how to do this.

File: doc/rose-install.html [hunk 4]
```diff
@@ -346,14 +350,156 @@ <h2 id="rose">Configuring Rose</h2>
...
+      <samp>https://somehost.on.the.internet/rosie/x</samp>, you can check that
+      you have access by typing the following on the command line <code>curl -I
+      https://somehost.on.the.internet/rosie/x</code>. It should return a HTTP
+      code 200. If you are prompted for a username and a password, you may need
+      to have access to a keyring to cache the authentication information.</li>
+
+      <li>You can access the Rosie web service using the Rosie client.  E.g.
+      using the above configuration for the prefix <em>x</em>, type the
+      following on the command line: <code>rosie hello --prefix=x</code>. It
+      should return a greeting.</li>
```
Comment (kaday) [inline doc/rose-install.html]: Give an example of the kind of greeting that they would expect to see returned.

**Comment by kaday** [comment]
Also, I know you did not change line 573 but would you mind changing "built in" to "built-in". Thanks.

**Comment by matthewrmshin** [comment]
Comments addressed.

**Comment by kaday** [comment]
Looks good to me.

**Pull request merged**